src/docker/superexec/README.md

```diff
@@ -1,3 +1,11 @@
+# Deprecation Notice
+
+Starting with Flower version 1.13.0, the `flwr/superexec` Docker image has been deprecated.
```

Comment:
```suggestion
Starting with Flower version 1.13.0, the `flwr/superexec` Docker image has been deprecated because the SuperExec has been removed in Flower 1.13 (to be precise: the functionalities of the SuperExec have been merged into the SuperLink).
```